README.md

```diff
@@ -111,3 +113,11 @@ More advanced TLS support using user-provided CAs or per-Materialize `Issuer`s a
 
 Due to limitations in Terraform, it cannot plan Kubernetes resources using CRDs that do not exist yet. We need to first install `cert-manager` in the first `terraform apply`, before defining any `ClusterIssuer` or `Certificate` resources which get created in the second `terraform apply`.
 <!-- END_TF_DOCS -->
+
+
+
+#### Storage Bucket Versioning
+By default storage bucket versinioning is turned off. This both reduces
```

Comment:
```suggestion
By default, storage bucket versioning is turned off. This both reduces
```

Also, added https://github.com/MaterializeInc/materialize/pull/32041 to go with this change.  I just used v0.2.1 as a placeholder ... but can change to whatever version.